### PR TITLE
feat: add unban command

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
I have seen this notification many times now, and, every time, I read it as `feat: add urban command`, and question what that could mean.

_sad, little tmi, but here we are_

File: src/commands/unban.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        );
+        return;
+      }
+
+      if (target.id === member.user.id) {
+        await interaction.editReply("You cannot unban yourself.");
+        return;
+      }
+      if (target.id === Bot.user?.id) {
+        await interaction.editReply("You cannot unban the bot.");
```
Comment (ShaunSHamilton) [inline src/commands/unban.ts:55]: Can you ban the bot?

Comment (naomi-lgbt) [reply src/commands/unban.ts:55]: You cannot ban the bot.

**Comment by naomi-lgbt** [comment]
> I have seen this notification many times now, and, every time, I read it as feat: add urban command, and question what that could mean.

The `urban` command allows you to search Urban Dictionary.

Also, the bot has been banned for Code of Conduct violations. 🤣

**Comment by Nirajn2311** [comment]
> Also, the bot has been banned for Code of Conduct violations. 

Why? What was being violated?

**Pull request merged**